src/commands/preview/mod.rs

```diff
@@ -57,47 +58,37 @@ pub fn preview(
 
         if !headless {
             open_browser(&format!(
-            "https://cloudflareworkers.com/?wrangler_session_id={0}&wrangler_ws_port={1}&hide_editor#{2}:{3}{4}",
-            &session.to_string(), ws_port, script_id, https_str, preview_host,
-        ))?;
+                "https://cloudflareworkers.com/?wrangler_session_id={0}&wrangler_ws_port={1}&hide_editor#{2}:{3}",
+                session, ws_port, script_id, browser_url
+            ))?;
+
+            message::preview("Your Worker is a Workers Site, please preview it in browser window.");
```

Comment:
This logic isn't quite right

```
        let msg = if sites_preview {	
            "Your Worker is a Workers Site, please preview it in browser window.".to_string()	
        } else {	
            format!("Your Worker responded with: {}", worker_res)	
        };
```

should be printing this output only if it's a workers site

Reply:
Presumably it came from a different version of the master when I pulled from upstream. Will patch that now.